src/bip47/address.cpp

```diff
@@ -0,0 +1,18 @@
+#include "bip47/address.h"
+#include "bip47/bip47utils.h"
+
+namespace bip47 {
+
+CAddress::CAddress(CBitcoinAddress const & address, bool isUsed)
+:address(address), isUsed_(isUsed)
+{}
+
+void CAddress::isUsed(bool isUsed) {
```

Comment:
woudln't be better to rename this setUseed or like that ?

Reply:
Fixed, thanks!